operator/src/main/resources/application.properties

```diff
@@ -2,6 +2,7 @@ agent.status.interval=60s
 strimzi.bundle.interval=60s
 strimzi.bundle.approval-delay=120s
 %test.strimzi.bundle.approval-delay=0s
+strimzi.clusterrolebinding-scan.interval=PT60M
```

Comment:
@rareddy default interval is 60 minutes ^

Reply:
oops I thought both mentioned on call and config property were different